src/services/homework/hooks/index.js

```diff
@@ -73,9 +73,6 @@ const isCourseHomeworkTeacher = (userId, homework) => {
 	if (!isValidId(userId)) throw new GeneralError('missing valid userId', { userId });
 	if (!homework || !homework.courseId) throw new GeneralError('course id (populated) in homework');
```

Comment:
shouldn't we check if homework.courseId is null first and then return false instead of throwing an error?